Automattic/sensei
# Separate settings for button alignment and text alignment

Closes https://github.com/Automattic/sensei/pull/3737

### Changes proposed in this Pull Request

* This PR separates settings for button alignment and the button text alignment.
  * It uses the original block alignment behavior, with a few hacks to make it work in some different situations, as it's an `inline-block` in the DOM.

Using the original block alignment behavior seems the most correct approach in this case. Anyway, I created a [different approach](https://github.com/Automattic/sensei/pull/3737), where the block is contained always in the same vertical space, just moving the button to the left, right, center, and making it full.

### Testing instructions

* Open the course editor.
* Add a "Take Course" and/or "Contact teacher" block.
* Play with the different block and text alignments.
* Save it and make sure it works properly in the frontend too.

**Observation:** It still depends on the theme templates research to make the full width work properly in the frontend. But if you wanna test, it's enough to remove the `<article>` and `<section>` tags introduced by the template.


### Screenshot / Video

![Oct-23-2020 15-13-57](https://user-images.githubusercontent.com/876340/97039215-729f4f00-1542-11eb-8ffe-bc4ded7a5909.gif)
